Tutorials/VPG_tutorial.md

```diff
@@ -1 +1,69 @@
 # Vanilla Policy gradient implementation
+
+
```

Comment:
Define the simulation environment briefly and then introduce what is VPG and all and mention the cost function that we need to optimize